scripts/ci/deploy.sh

```diff
@@ -19,18 +19,37 @@
 set -e
```

Comment:
The same script is able to deploy to both snapshot or production, depending on the environment.